src/app/oauth/components/oauth.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="oauth-home">
+    <div>
+        Your Google Access Token :
+        <span class="information">{{accessToken}}</span>
```

Comment:
This page is good... 
May be we show this as a tooltip / on-click tooltip to user when user clicks on "Signed-In User" in the toolbar icon as suggested elsewhere.